src/___new___/components/FelixConfigTable/index.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+
+const TableCRD = ({ fieldData }) => (
+  <table>
+    <thead>
+      <tr>
+        <th>Attribute</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+      <td>Key</td>
+      <td><code>{fieldData.NameYAML || 'No Default Value'}</code></td>
+      </tr>
+      <tr>
+        <td>Description</td>
+        <td dangerouslySetInnerHTML={{ __html: fieldData.DescriptionHTML }} />
```

Comment:
You should probably sanitize this html to be safe
https://blog.logrocket.com/using-dangerouslysetinnerhtml-react-application/

Reply:
Done.